sys/include/net/gnrc/netif/conf.h

```diff
@@ -111,7 +123,8 @@ extern "C" {
  * Default: 2 (1 link-local + 1 global address)
  */
 #ifndef CONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF
-#define CONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF    (2)
+#define CONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF    (2 + \
+                                               GNRC_NETIF_IPV6_DHCPV6_IA_NA_ADDR)
```

Comment:
Then again (and sorry if I contradict past-me in that case, if so, please ignore): why have the indirection via `GNRC_NETIF_IPV6_DHCPV6_IA_NA_ADDR` in the first place. Just use

```suggestion
                                               CONFIG_DHCPV6_CLIENT_ADDR_LEASE_MAX)
```

here directly.

Reply:
This should now be fixed. Maybe some details have to be ironed out, however.